akka-http-core/src/main/java/akka/http/javadsl/model/HttpMessage.java

```diff
@@ -55,6 +58,11 @@
      */
     ResponseEntity entity();
 
+    /**
+     *  Drains entity stream of this message
+     */
+    CompletionStage<Done> ignoreDataBytesJ(Materializer materializer);
```

Comment:
> They would both have to be discardDataBytes.

If so, then we're forced to use the implicit conversions trick explained in: 
a) https://github.com/akka/akka/pull/20707/files/a7d89d19a3dfda5e299a1a9b2714d44425e91b61#r65893253
Do you think it's worth it?

Otherwise we can't since it's the same signature, and different return types.

We could try b) `I also had an idea of ignoreDataBytes().future / ignoreDataBytes().completionStage?`